Sail/src/Sail/entities/systems/input/GameInputSystem.cpp

```diff
@@ -1,19 +1,187 @@
 #include "pch.h"
 #include "GameInputSystem.h"
-#include "..//..//..//api/Input.h"
+#include "Sail/graphics/camera/CameraController.h"
+#include "Sail/entities/components/Components.h"
+#include "../../../api/Input.h"
+#include "Sail/KeyBinds.h"
 
 GameInputSystem::GameInputSystem() {
-	
+	m_yaw = 90.f;
+	m_pitch = 0.f;
+	m_roll = 0.f;
```

Comment:
No required components? This will cause all entities to be added to the system. Create a PlayerComponent that acts as a flag?